tests/phpunit/CRM/Contribute/BAO/ContributionPageTest.php

```diff
@@ -15,35 +15,6 @@
  */
 class CRM_Contribute_BAO_ContributionPageTest extends CiviUnitTestCase {
 
-  /**
-   * Create() method (create Contribution Page)
-   */
-  public function testCreate(): void {
-
-    $params = [
-      'qfkey' => '9a3ef3c08879ad4c8c109b21c583400e',
-      'title' => 'Test Contribution Page',
-      'financial_type_id' => 1,
-      'intro_text' => '',
-      'footer_text' => 'Thanks',
-      'is_for_organization' => 0,
-      'for_organization' => ' I am contributing on behalf of an organization',
-      'goal_amount' => '400',
-      'is_active' => 1,
-      'honor_block_title' => '',
-      'honor_block_text' => '',
-      'start_date' => '20091022105900',
-      'start_date_time' => '10:59AM',
-      'end_date' => '19700101000000',
-      'end_date_time' => '',
-      'is_credit_card_only' => '',
-    ];
-
-    $contributionPageID = CRM_Contribute_BAO_ContributionPage::create($params)->id;
-    $this->assertIsInt($contributionPageID);
-    $this->callAPISuccess('ContributionPage', 'delete', ['id' => $contributionPageID]);
```

Comment:
It's hard to think of a more useless test

Reply:
I agree that just checking that id is an integer is silly, but the test would still have been useful as a sanity check to expose some typo or undefined var that gets introduced, or other error. But if writeRecord is tested with ContributionPage then there's no need for this now that create is deprecated.

Reply:
Sometimes different params have different code paths or trigger different errors. But overall I'm agreeing with you.